docs/api/main-config-core.md

```diff
@@ -0,0 +1,106 @@
+---
+title: 'config.core'
+---
+
+Type:
+
+```ts
+{
+  builder?: string | { name: string; options?: BuilderOptions };
+  channelOptions?: ChannelOptions;
+  crossOriginIsolated?: boolean;
+  disableProjectJson?: boolean;
+  disableTelemetry?: boolean;
+  disableWebpackDefaults?: boolean;
+  enableCrashReports?: boolean;
+  renderer?: RendererName;
+}
+```
+
+Configures Storybook's internal features.
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/main-config-core.js.mdx',
+    'common/main-config-core.ts.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+## `config.core.builder`
```

Comment:
I'm not sure about including the full object path in the header like this. I do the same in the title, and in the list of properties on the Overview page.

I tried without it, e.g. the title of this page would `core` and this heading would be `builder`, but it seemed to lack context.

Curious for others' thoughts.

Reply:
I decided to simplify the headings to only include the property name itself, e.g. this one is now `builder`.

To help with context, I added this between the page title (`core`, in this example) and the `Type`:

```
Parent: [main.js|ts configuration](./Overview.md)
```